docs/app/javascript/packs/validstate/Validstate.js

```diff
@@ -632,4 +632,13 @@ export default class Validstate {
     return /^(?:\+?1[-\s]?)?\(?([2-9][0-8][0-9])\)?[-\s]?([2-9]\d{2})[-\s]?(\d{4})$/.test(phoneNumber);
   }
 
+  /*
+  * @function elementOf
+  * @description Iterates over an array and checks if a given value is an element of the array
+  * @parameter value, array
+  * @return Boolean
+  */
+  elementOf(value, arr) {
```

Comment:
@anthonyjruffa this looks good. My only suggestion would be to normalize the strings to lowercase before checking. 

```
['Foo', 'Bar'].includes('foo') => false
['Foo', 'Bar'].includes('Foo') => true
```

Do you see a use case where we would want to be case sensitive?

Reply:
@CodyMcMichael is there a case where someone might have an array be something like `["foo", "Foo", "Bar"]`? If that case won't happen, then we can normalize it per your suggestion.

Reply:
@anthonyjruffa  Yeah you're right.. No I can't guarantee what a user's test cases will look like, so in that case the burden falls on them to know that and test for the correct things.

"One other option would be to have a way to let the user choose to make it case sensitive or not, and we can set the default to case insensitive."

I really like this idea, I've been pondering a way to add an options argument to the validations and for specific ones allow different configs to be set. This is a perfect use case for it. It would require a decent but of work on validstate core to support that.

I think for this case, what you have is good to merge, feel free to open up an issue describing the options use case and if you want to work on that that would be great!

Reply:
@CodyMcMichael thanks for your reply and thoughts! I can't guarantee I'll be able to make the time to work on that, but I just opened a new issue for it to allow someone else who might have more availability to pick it up.